platform/ui/src/components/HotkeysPreferences/hotkeysValidators.js

```diff
@@ -46,7 +47,7 @@ const conflictingValidator = ({ commandName, pressedKeys, hotkeys }) => {
 
   if (conflictingCommand) {
     return {
-      error: `"${conflictingCommand.label}" is already using the "${pressedKeys}" shortcut.`,
+      error: i18n.t('HotkeysValidators:Hotkey is already in use', {action: conflictingCommand.label, pressedKeys: pressedKeys }),
```

Comment:
Here's a bit tricky. Default fallback en-us -> key_name
If there is no "Hotkey is already in use" in en-us, the message will be "Hotkey is already in use"
Otherwise, it will be "{{pressedKeys}} already used in "{{action}}", e.g. "Z already used in Zoom"

Reply:
I love the fact that you used interpolation here and en-us should always have the key so I think we are fine. However, in the HotkeysValidator.json, I think it is best to keep the wording of the message the same as what it used to be.

Reply:
Could you then tell me what is the best way to use i18n here? Declare several keys like "is already using the" and "shortcut"?

Reply:
I am not sure I understand your question. Perhaps let me clarify that I believe you are using i18n and interpolation correctly here. I was simply suggesting that the entry in the en-us for `Hotkey is already in use` should be...

`"Hotkey is already in use": "{{action}} is already using the  \"{{processedKeys}}\" shortcut.",`

so that the wording matches what was there before. Does this make sense?